cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -61,4 +66,26 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
+      - name: healthz
+        image: gcr.io/google_containers/healthz-server:1.0
+        resources:
+          limits:
+            cpu: 100m
+            memory: 5Mi
```

Comment:
4MB is minimal you can specified, otherwise docker would reject it. But I am not comfortable for less than 10M. :-)